masterlist.yaml

```diff
@@ -9774,3 +9774,54 @@ plugins:
         util: 'SSEEdit v4.0.1n RC4 for 4.0.2'
       - crc: 0xDA25CBFC
         util: 'SSEEdit v4.0.1n RC4 for 4.0.2'
+  - name: 'MissingEZs_AllExteriors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/23609'
+        name: 'Missing Encounter Zones Fixed on Nexus Mods'
+    group: *highPriorityGroup
+    after:
+      - 'RelightingSkyrim_SSE.esp'
+      - 'Populated Dungns Caves Ruins Legendary.esp'
+    tag:
+      - C.Encounter
+      - C.Location
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Realistic Water Two' ]
+        condition: 'active("RealisticWaterTwo.esp") and not active("MZEF_ExteriorsRW2Patch.esp")'
+  - name: 'MZEF_ExteriorsRW2Patch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/23609'
+        name: 'Missing Encounter Zones Fixed on Nexus Mods'
+    after:
+      - 'RealisticWaterTwo.esp'
```

Comment:
You don't need to add an `after` if the plugin is a master.
![image](https://user-images.githubusercontent.com/1944639/53516978-7bb5ef80-3ac5-11e9-9245-a49e98659653.png)

Reply:
good to know! should I change that and re-commit/pull?

Reply:
Yes you can remove that after rule for the patch.
Just make another commit to this pull request.